extension/docs/Refund.md

```diff
@@ -181,6 +181,12 @@ Follow these recommendations when using the idempotency key:
 - Generate idempotency keys using the version 4 (random) UUID type to prevent two API credentials under the same account from accessing each others responses.
 - Use [exponential backoff](https://en.wikipedia.org/wiki/Exponential_backoff) when retrying.
 
+#### Generating idempotency key by adyen-integration
+
+adyen-integration can automatically generate unique idempotency key for refund requests. It uses transaction ID as the idempotency key.
+
+In order to use this feature, set the option [`generateIdempotencyKey=true`](./HowToRun.md#optional-attributes). For every refund request adyen-integration will add an idempotency key if the custom field `idempotencyKey` is not present. If the custom field `idempotencyKey` is present, its value will be taken as the idempotency key.
+
```

Comment:
I think you can include following Adyen documentation as same as `ManualCapture.md`

> For in-depth information about API idempotency in Adyen, see [Adyen's documentation](https://docs.adyen.com/development-resources/api-idempotency).

Reply:
I had it here like you suggested but I moved it to the bottom. The reason is when the link is here it looks like the user has to check the docs to use the Refund. That is not true and the documentation is rather a source where we got the info. User does not have to care about it at all. Therefore I moved to the bottom so it's not confusing.

Reply:
ok i found it now. It is not in this pr so i didn't realize you added it.